src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -48,7 +49,7 @@ type Props = TravelDetailsMapScreenParams & {
 };
 
 const FOLLOW_ZOOM_LEVEL = 14.5;
-const FOLLOW_MIN_ZOOM_LEVEL = 8;
+const FOLLOW_MIN_ZOOM_LEVEL = 6; // keep below 6.32 to avoid mapbox bug
```

Comment:
As we discussed it seems more like the values for Zoom are not so accurate when listening for camera changes, but for other places like by example `onMapIdle` are correct. There is a strange behaviour that shows 6.32 as map zoom as the first value (I think that must be the default value for Mapbox) but then it changes in a later state when a location is updated, so there is something else going on here. But yes, this might be the shortest solution for now, and take a look in further updates on MapBox.

Reply:
I think we should really try to keep the state correct. With the solution now we "accept" that the faulty value 6.32 comes in the state, but it is circumvented by changing this `FOLLOW_MIN_ZOOM_LEVEL`. Decided together with Marius that we could look at if the value from `onMapIdle` is more correct.

Reply:
I also agree that `onMapIdle` is the better solution, separating the heading and zoom. Yesterday we tried this, but then @marius-at-atb had a talk with the designers and QA. So I also opt for this option!

Reply:
Now using onMapIdle with the latest commit 👍